### ISSUE TITLE
Example tests not working

### ISSUE THREAD

**Issue by rainerhahnekamp** [testing-library/angular-testing-library#139]
I just cloned the git repo and tried to run the example tests locally.

Run following commands:
```
yarn --frozen-lockfile
yarn nx test
```

All tests fail. That's what I get for the first one and more or less the same for the rest.
```
 FAIL   Example  apps/example-app/app/examples/02-input-output.spec.ts
  ● Test suite failed to run

    projects/testing-library/src/lib/testing-library.ts:306:41 - error TS2591: Cannot find name 'process'. Do you need to install type definitions for node? Try `npm i @types/node` and then add `node` to the types field in your tsconfig.

    306 if (typeof afterEach === 'function' && !process.env.ATL_SKIP_AUTO_CLEANUP) {
                                                ~~~~~~~
```

**Comment by timdeschryver** [comment]
Hey, thanks! I don't know what happened but I also had this experience while working on a new cloned version of this repo. I'm working on a fix in #138

**Comment by timdeschryver** [comment]
That PR is now merged, feel free to try again and if it fails you can reopen this issue.
